Expose contactedReviewers API in v1.0 and misc. updates (#15239)
microsoftgraph/microsoft-graph-docs: api-reference/v1.0/resources/accessreviewinstancedecisionitemserviceprincipalresource.md

> ---

Reviewer: Please sync the details in this file with the details in the beta file.

title: "accessReviewInstanceDecisionItemServicePrincipalResource resource type"
description: "Represents assignment to service principals. "
author: "isabelleatmsft"
ms.localizationpriority: medium
ms.prod: "governance"
doc_type: resourcePageType
---

# accessReviewInstanceDecisionItemServicePrincipalResource resource type

Namespace: microsoft.graph

Represents assignment to service principals. accessReviewInstanceDecisionItemServicePrincipalResource is an open type that allows for other properties to be passed in.

Inherits from [accessReviewInstanceDecisionItemResource](accessreviewinstancedecisionItemresource.md).

## Properties
|Property|Type|Description|
|:---|:---|:---|
| appId | String | The globally unique identifier of the application to which access has been granted. |

> | displayName | String | Display name of the resource. Inherited from [accessReviewInstanceDecisionItemResource](accessreviewinstancedecisionItemresource.md).|

Reviewer 2: ```suggestion
| displayName | String | Display name of the service principal. Inherited from [accessReviewInstanceDecisionItemResource](accessreviewinstancedecisionItemresource.md).|
```

| id | String | Identifier of the decision item resource. Inherited from [accessReviewInstanceDecisionItemResource](accessreviewinstancedecisionItemresource.md). |

> | type | String | Type of resource. Types include: `Group`, `ServicePrincipal`, `DirectoryRole`, `AzureRole`, `AccessPackageAssignmentPolicy`.  Inherited from [accessReviewInstanceDecisionItemResource](accessreviewinstancedecisionItemresource.md). |

Reviewer 2: ```suggestion
| type | String | Type of resource. Type will always be:  `ServicePrincipal`.  Inherited from [accessReviewInstanceDecisionItemResource](accessreviewinstancedecisionItemresource.md). |
```
